src/OpenTelemetry.Exporter.Zipkin/Implementation/ZipkinActivityConversionExtensions.cs

```diff
@@ -208,10 +208,8 @@ internal static bool ProcessTags(ref AttributeEnumerationState state, KeyValuePa
                 {
                     state.ServiceNamespace = strVal;
                 }
-                else
-                {
-                    PooledList<KeyValuePair<string, object>>.Add(ref state.Tags, new KeyValuePair<string, object>(key, strVal));
-                }
+
+                PooledList<KeyValuePair<string, object>>.Add(ref state.Tags, new KeyValuePair<string, object>(key, strVal));
```

Comment:
@eddynaka Can you also update the code right below this? Something like:

```csharp
            else
            {
                if (attribute.Value is int intVal && attribute.Key == SemanticConventions.AttributeNetPeerPort)
                {
                   state.Port = intVal;
                }
                
                PooledList<KeyValuePair<string, object>>.Add(ref state.Tags, attribute);	                
            }
```

That way "int" port is also sent.

Reply:
thanks for the guidance! just updated!